postgres/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "coredb"
-version = "15.1.0-coredb.11"
+version = "15.2-coredb.1"
```

Comment:
we're running postgres 15.2 now

Reply:
ah I think it requires to add patch version or just add publish = false